misc.Rmd

```diff
@@ -65,69 +80,45 @@ citation()
 Calling it with a package name tells you how to cite that package:
 
 ```{r}
-citation("lubridate")
+citation("tidyverse")
 ```
 
-To customise the citation for your package, add a `inst/CITATION` that looks like this:
+The associated `inst/CITATION` file looks like this:
 
 ```{r, echo = FALSE, comment = ""}
-citation <- readLines(system.file("CITATION", package = "lubridate"))
+citation <- readLines(system.file("CITATION", package = "tidyverse"))
 cat(citation, sep = "\n")
 ```
 
-You need to create `inst/CITATION`.
-As you can see, it's pretty simple: you only need to learn one new function, `citEntry()`.
-The most important arguments are:
-
--   `entry`: the type of citation, "Article", "Book", "PhDThesis" etc.
-
--   The standard bibliographic information like `title`, `author` (which should be a `personList()`), `year`, `journal`, `volume`, `issue`, `pages`, ...
-
-A complete list of arguments can be found in `?bibentry`.
-
-Use `citHeader()` and `citFooter()` to add additional exhortations.
-
-### Other languages {#sec-inst-other-langs}
-
-Sometimes a package contains useful supplementary scripts in other programming languages.
-Generally, you should avoid these, because it adds an additional extra dependency, but it may be useful when wrapping substantial amounts of code from another language.
-For example, [gdata](https://cran.r-project.org/web/packages/gdata/index.html) wraps the Perl module [Spreadsheet::ParseExcel](https://search.cpan.org/~dougw/Spreadsheet-ParseExcel-0.65/) to read excel files into R.
-
-The convention is to put scripts of this nature into a subdirectory of `inst/`, `inst/python`, `inst/perl`, `inst/ruby` etc.
-If these scripts are essential to your package, make sure you also add the appropriate programming language to the `SystemRequirements` field in the `DESCRIPTION`.
-(This field is for human reading so don't worry about exactly how you specify it.)
+You can call `usethis::use_citation()` to initiate this file and fill in your details.
+Read the `?bibentry` help topic for more details.
 
-Java is a special case and the best place to learn more is the documentation of the rJava package (<http://www.rforge.net/rJava/>).
+## Configuration tools
 
-## Demos {#demo}
+If a package has a configuration script (`configure` on Unix-alikes, `configure.win` on Windows), it is executed as the first step by `R CMD  INSTALL`. And if the configuration script needs auxiliary files, those should be located in the `tools/` directory.
+The scripts below `tools/` can have an effect on the installed package, but the contents of `tools/` will not ultimately be present in the installed package.
+In any case, this is mostly (but not solely) relevant to packages containing compiled code, which is beyond the scope of this book.
 
-A demo is an `.R` file that lives in `demo/`.
```

Comment:
I beefed up the introduction's bullet point on `demo/`, then deleted any further content about it.